test/pipelines/pipelines-it-local.yml

```diff
@@ -21,6 +21,12 @@ jobs:
       PATH=$HOME/.local/bin:$PATH nnictl package install --name=BOHB
       PATH=$HOME/.local/bin:$PATH nnictl package install --name=PPOTuner
     displayName: 'Install dependencies for integration tests'
+  - script: |
+      set -e
+      cd examples/tuners/customized_tuner
+      python3 setup.py develop
```

Comment:
Please check this error log:  https://msrasrg.visualstudio.com/NNIOpenSource/_build/results?buildId=14256&view=logs&j=75aeeae9-dab9-55a1-b144-0ab48bfd0d00&t=3f120a8d-7233-5989-6cfa-cd23207edde0

It seems you need to add `--user` here